README.md

```diff
@@ -585,26 +698,37 @@ A UCAN is valid inclusive from the `nbf` time and until the `exp` field. If the
 
 In delegation, the `aud` field of every proof MUST match the `iss` field of the outer UCAN (the one being delegated to). This alignment MUST form a chain back to the originating principal for each resource.
```

Comment:
Related to my prev comment: does this hold if the `iss` is a did that points to a key with the hash fragment?

Reply:
Yup! The hash fragment is just there to help figure out the specific key that signed, which is different from which agent is being delegated to. Will add clarifying text 👍